### ISSUE TITLE
Unable to install packages with Anaconda index behind a proxy with proper settings on Windows

### ISSUE THREAD

**Issue by Juanlu001** [ContinuumIO/anaconda-issues#553]
I have proper proxy settings and a working Anaconda installation:

```
C:\Users\<USER>>pip install pep8 -d . --no-cache-dir --proxy https://<USER>:<PASS>@<HOST>:80 -i https://pypi.python.org/simple
Collecting pep8
  Downloading pep8-1.6.2-py2.py3-none-any.whl (40kB)
    100% |################################| 40kB 883kB/s
  Saved c:\users\<USER>\pep8-1.6.2-py2.py3-none-any.whl
Successfully downloaded pep8
```

But when requesting Anaconda packages, these same settings fail:

```
C:\Users\<USER>>pip install pep8 -d . --no-cache-dir --proxy https://<USER>:<PASS>@<HOST>:80 -i https://pypi.anaconda.org/pypi/simple
Collecting pep8
  Retrying (Retry(total=4, connect=None, read=None, redirect=None)) after connection broken by 'ProxyError('Cannot connect to proxy.', OSError('Tunnel connection failed: 403 Forbidden',))': /pypi/simple/pep8/
  Retrying (Retry(total=3, connect=None, read=None, redirect=None)) after connection broken by 'ProxyError('Cannot connect to proxy.', OSError('Tunnel connection failed: 403 Forbidden',))': /pypi/simple/pep8/
Operation cancelled by user
```

Besides, conda fails to reach the urls:

```
C:\Users\<USER>>conda install pep8
Fetching package metadata: Could not connect to https://repo.continuum.io/pkgs/pro/noarch/
.Could not connect to https://repo.continuum.io/pkgs/free/noarch/
.Could not connect to https://repo.continuum.io/pkgs/pro/win-64/
.Could not connect to https://repo.continuum.io/pkgs/free/win-64/
.
Solving package specifications: ................
# All requested packages already installed.
# packages in environment at C:\Users\<USER>\AppData\Local\Continuum\Anaconda3:
#
pep8                      1.6.2                    py35_0

C:\Users\<USER>>type .condarc
proxy_servers:
  https: https://<USER>:<PASS>@<HOST>:80
```

**Comment by Juanlu001** [comment]
The problem was with the company proxy, which blacklists certain urls based on user agent, similar to https://github.com/pypa/pip/issues/1565. I can easily patch conda to suit my needs.

**Comment by nishankmahore** [comment]
Install MiniConda
sh Miniconda2-latest-Linux-x86_64.sh -f
In the same folder of previous  conda
It will solve all conad related problem

Works for me as charm

**Comment by mingwandroid** [comment]
Please don't install Miniconda2-latest-Linux-x86_64.sh on Windows.

**Comment by willycs40** [comment]
I had a similar issue to this. I was trying to make conda work behind a corporate firewall with anonymous login. 
- I set environment variables, and also set .condarc settings. 
- wget worked, pip worked
- conda gave me proxy error. 
- eventually, in desperation, I tried changing the proxy from http://myproxy.company.com:8888 to http://guest:guest@myproxy.company:8888 (i.e. I added made up credentials) and it started working.

It appears conda really wants credentials, even when it doesn't need them.
